js/friday.js

```diff
@@ -1,34 +1,38 @@
 (function IsItFridayYet() {
   const date = new Date();
```

Comment:
In my constant struggle with naming things, we are going to tweak some of these constants.